py-rse/configuration.Rmd

```diff
@@ -51,162 +53,361 @@ fonts:
 - Serif
 ```
 
-And here's an extension to our script template
-that uses `argparse` and provide
-a default configuration that the user can override:
+## Matplotlib configuration {#py-rse-configuration-matplotlib}
+
+To see overlay configuration in action,
+let's consider one of the most common tasks in data science --
+changing the size of the labels on a plot.
+The size of the labels on our Jane Eyre plot
+is fine for viewing on our screen,
+but they'll need to be bigger
+if we want to include the figure in a powerpoint presentation or report.
+
+```{r py-rse-configuration-jane-eyre-default, echo=FALSE, fig.cap="Word frequency distribution for the book Jane Eyre with default label sizes."}
+knitr::include_graphics("figures/py-rse-configuration/jane_eyre_default.png")
+```
+
+In order to configure the label sizes,
+we could use any of the overlay options described above:
+
+-  Edit the system-wide matplotlib configuration YAML file known as the `matplotlibrc` file
+-  Create a user-specific matplotlib style sheet (which also uses the YAML format) 
+-  Create a job-specific YAML file that the `plotcounts.py` script can parse
+-  Add some new command-line options to `plotcounts.py` 
+
+Let's consider these options one at a time.
+
+### Edit the matplotlibrc file
```

Comment:
maybe mentioned that `rc` stood (once upon a very long time ago) for "resource control"?

Reply:
Wasn't it "run commands"? https://en.wikipedia.org/wiki/Run_commands Or is "resource control" even older?

Reply:
There were a few different takes on what `rc` has historically stood for at [this stack overflow question](https://stackoverflow.com/questions/37728087/rc-params-in-matplotlib-what-does-rc-stand-for), so I was reluctant to go beyond simply calling it "runtime configuration."